### PR TITLE
Ed25519 testcase

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: apps/examples/security_test/seclink/sl_key_test.c [hunk 1]
```diff
@@ -130,11 +130,30 @@ END_TESTCASE
 /*
  * Desc: set asymmetric key
  */
+char key_a_buf[1024];
```
Comment (jeongchanKim) [inline apps/examples/security_test/seclink/sl_key_test.c:133]: How about adding `static`?

Comment (pillip8282) [reply apps/examples/security_test/seclink/sl_key_test.c:133]: Thanks It's better. I'll modify it

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**